server/routes/events.js

```diff
@@ -30,15 +31,18 @@ router.post('/:id/attending', (req, res) => {
     })
 })
 
-router.get('/:id/attending', (req, res) => {
-  const { id } = req.params
-  getEventsForUser(Number(id))
-    .then((events) => {
-      res.status(200).json(events)
-    })
-    .catch(() => res.status(500).send('DATABASE ERROR'))
+router.get('/:id/attending', getTokenDecoder(), (req, res) => {
+  if (req.user) {
+    const { id } = req.params
+    getEventsForUser(Number(id))
+      .then((events) => {
+        res.status(200).json(events)
+      })
+      .catch(() => res.status(500).send('DATABASE ERROR'))
+  } else {
+    res.status(500).send('authentication token not provided')
```

Comment:
```suggestion
    res.status(401).send('authentication token not provided')
```

❔ https://en.wikipedia.org/wiki/List_of_HTTP_status_codes#401

Reply:
I have updated all the status response for the events route